src/main/frontend/components/settings.cljs

```diff
@@ -1018,11 +1010,7 @@
          (when enable-sync?
            (sync-diff-merge-switcher-row enable-sync-diff-merge?))
          [:div.text-sm
-          (t :settings-page/sync-desc-1)
```

Comment:
We should be passing in links as an argument but unfortunately we can't. Please revert these changes for reasons mentioned in `:on-boarding/android-intro-dropsync` and add the TODO